Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -83,22 +87,19 @@ void MainWindow::cleanup() {
     m_viewer->getGizmoManager()->cleanup();
 }
 
+// Connection to gizmos must be done after GL is initialized
```

Comment:
Can you just tell me the reason why?

Reply:
Quickly : because of bad design (at least perfectible design)
More slowly, because the constructors of gizmos use resources from Engine that are only available after OpenGL were initialized. This is a consequence of gizmo creation process that meld visual and semantic representation of gizmos, keymapping registration that need to access the gizmos and strange event ordering of Qt that depends on the way the application initialize its resources, windows, ui, ....